instrumentation/log4j/log4j-2-testing/src/main/groovy/Log4j2Test.groovy

```diff
@@ -14,24 +14,22 @@
  * limitations under the License.
  */
 
-package io.opentelemetry.instrumentation.log4j.v2_13_2
-
+import io.opentelemetry.auto.test.AgentTestRunner
 import io.opentelemetry.auto.test.utils.TraceUtils
+import io.opentelemetry.instrumentation.log4j.v2_13_2.ListAppender
 import io.opentelemetry.trace.Span
 import io.opentelemetry.trace.TracingContextUtils
 import org.apache.logging.log4j.LogManager
-import org.apache.logging.log4j.Logger
-import spock.lang.Specification
-
-class Log4j2Test extends Specification {
-
-  private static final Logger logger = LogManager.getLogger("TestLogger")
 
+abstract class Log4j2Test extends AgentTestRunner {
```

Comment:
If I'm reading correctly, this is the base for both library and auto instrumentation. In that case it can't extend `AgentTestRunner` which executes auto instrumentation. We `AgentTestTrait` and `InstrumentationTestTrait` instead, do those work?

Reply:
It shouldn't execute auto instrumentation because it's not on classpath, but using those traits is a good idea anyway -- no need to run the agent if it's unnecessary. I'll fix that.